jenkins/job-configs/kubernetes-jenkins-pull/credentials.yaml

```diff
@@ -0,0 +1,7 @@
+- wrapper:
+    name: e2e-credentials-binding  # TODO(fejta): make this match non-pull
```

Comment:
Jenkins only generates random IDs if you decline to choose a value for the ID when you create it